File: java/program83.java
import java.io.*;

class program83
{
    public static int CountDigit(int No)
    {
        int iCnt = 0;

        while(No != 0)
        {
            iCnt++;
            No = No / 10;
        }
        return iCnt;
    }

    public static void main(String arg[])
    {
        BufferedReader bobj = new BufferedReader(new InputStreamReader(System.in));
        int value = 0;

        try
        {
            System.out.println("Enter number :");
            value = Integer.parseInt(bobj.readLine());
        }
        catch(IOException eobj)
        {
            
        }
        finally
        {
            int ret = CountDigit(value);
            System.out.println("Count of the Digits is : "+ret);
        }
    }
}

File: cpp/program83.cpp
// program49.c

#include<iostream>
using namespace std;

int CountDigits(int iNo)
{
    int iCnt = 0;

    while(iNo != 0)
    {
        iCnt++;
        iNo = iNo / 10;
    }
    return iCnt;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    cout<<"Enter number \n";
    cin>>iValue;

    iRet = CountDigits(iValue);
    cout<<"Count of the Digits is : "<<iRet<<"\n";

    return 0;
}